src/mavlink_receive.c

```diff
@@ -212,10 +213,9 @@ static void handle_mavlink_message(void)
 // function definitions
 void mavlink_receive(void)
 {
-  while (serialTotalBytesWaiting(Serial1))
+  while (serial_bytes_available())
```

Comment:
Do we want to every support more than one UART?  For example, both a USB connection and a serial radio?  Or does that get into feature creep?